apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/layout.tsx

```diff
@@ -59,116 +64,120 @@ export default async function ChainPageLayout({
   ];
 
   return (
-    <div className="container flex flex-row h-full">
-      <Sidebar
-        header={
-          <div className="flex items-center gap-1.5 mb-5 flex-wrap">
-            <Link
-              href="/chainlist"
-              className="text-muted-foreground hover:text-foreground text-sm"
-            >
-              Chainlist
-            </Link>
-            <span className="text-muted-foreground text-sm"> / </span>
-            <Link href={`/${chain.slug}`} className="text-foreground text-sm">
-              {chain.name.replace("Mainnet", "")}
-            </Link>
-          </div>
-        }
-        links={sidebarLinks}
-      />
-
-      <div className="flex flex-col w-full pb-10">
-        {/* Icon + Background */}
-        <ChainHeader
-          headerImageUrl={chainMetadata?.headerImgUrl}
-          logoUrl={chain.icon?.url}
-          chain={chain}
-        />
-
-        <div className="flex lg:hidden justify-end">
-          <MobileSidebar
-            links={sidebarLinks}
-            trigger={
-              <Button size="icon" variant="outline">
-                <MenuIcon strokeWidth={1} />
-              </Button>
-            }
-            footer={
-              <div className="mt-5 border-t pt-6 pb-2">
-                <Link
-                  href="/chainlist"
-                  className="px-2 text-sm flex justify-start gap-2 items-center text-muted-foreground hover:text-foreground"
-                >
-                  <ArrowLeftIcon className="size-4" />
-                  View all chains
-                </Link>
-              </div>
-            }
-          />
+    <>
+      <div className="border-b border-border h-14 pl-7 flex">
+        <div className="my-auto flex flex-row gap-3 text-muted-foreground">
+          <Link href="/chainlist" className="hover:text-foreground">
+            Chainlist
+          </Link>
+          <ChevronRight className="size-4 my-auto" />
+          <Link
+            href={chain.testnet ? "/chainlist/testnet" : "/chainlist/mainnet"}
+            className="hover:text-foreground"
+          >
+            {chain.testnet ? "Testnet" : "Mainnet"}
+          </Link>
+          <ChevronRight className="size-4 my-auto" />
+          <Link href={`/${chain.slug}`} className="hover:text-foreground">
+            {chain.name.replace("Mainnet", "")}
+          </Link>
         </div>
+      </div>
```

Comment:
prefer using `shadcn -> breadcrumb`: https://ui.shadcn.com/docs/components/breadcrumb

Reply:
Didn't think of that... anyway done!